### ISSUE TITLE
Inventory List: Labels -- Adding an existing label and checking "Show in List" does not result in label being displayed in list

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#3833]
Instance: production
SHA: e5fd20aee

In the Inventory List, I want to add an existing label to a set of records, and I want that label to show up in the list view, so I check the "Show in List" option at the top of the screen, which is active. 

![image](https://user-images.githubusercontent.com/6314950/217086755-963449d9-57fe-4e9b-93f9-ae00afdd2c98.png)

However, checking that box doesn't seem to work for a label that already exists in terms of showing it in the Inventory List view. After applying that label, I look in the Inventory List but it is not showing up. 

![image](https://user-images.githubusercontent.com/6314950/217087776-8d890a05-1736-4e53-bafd-d5feaa123130.png)

**Comment by RDmitchell** [comment]
hmmm, now I can't reproduce this problem ... sigh

**Comment by anchapin** [comment]
@RDmitchell, that check box is only for new labels. As far as I know, you have to go to the Labels page in organization settings to edit existing labels and make them show in the list.

**Comment by anchapin** [comment]
I agree with you. I'll edit this issue to indicate that the add/remove labels action model should include "Show In List" checkbox for each of the existing labels.

**Comment by RDmitchell** [comment]
@anchapin -- great thanks !!

**Comment by isalanglois** [comment]
@RDmitchell - marking for FY24 development
